src/ldf-client-ui.js

```diff
@@ -4,7 +4,7 @@
 // This exports the webpacked jQuery.
 window.jQuery = require('../deps/jquery-2.1.0.js');
 var N3 = require('n3');
-
+window.comunicaVersion = require('../node_modules/@comunica/actor-init-sparql/package.json').version;
```

Comment:
Instead of setting a global value, I'd just update `"comunica-version"` from this file directly.

Reply:
Also, could you add the following TODO above this line?
`// TODO: Dynamically expose the version in ActorInitSparql so that we can retrieve it from there instead of relying on package.json reading.`